README.md

```diff
@@ -1,3 +1,143 @@
+
+fdk_extension_client.application_proxy_routes_bp.add_route(test_route_handler, "/1234")
+
+app.add_route(webhook_handler, "/webhook")
+
+app.blueprint(fdk_extension_client.platform_api_routes_bp)
+app.blueprint(fdk_extension_client.application_proxy_routes_bp)
+
+# debug logs enabled with debug = True
+app.run(host="0.0.0.0", port=8000, debug=True)
+
+```
+
+#### How to call platform apis?
+
+To call platform api you need to have instance of `PlatformClient`. Instance holds methods for SDK classes. All routes registered under `platform_api_routes_bp` blueprint will have `platform_client` under request context object which is instance of `PlatformClient`.
+
+> Here `platform_api_routes_bp` has middleware attached which allows passing such request which are called after launching extension under any company.
+
+```python
+async def test_route_handler(request):
+    try:
+        data = await request.conn_info.ctx.platform_client.lead.getTicket(id="61b08ec5c63045521bcf124f")
+        return response.json({"data": data["json"]})
+    except Exception as e:
+        return response.json({"error_message": str(e)}, 500)
+        
+fdk_extension_client.platform_api_routes_bp.add_route(test_route_handler, "/test/routes")
+app.blueprint(fdk_extension_client.platform_api_routes_bp)
+```
+
+#### How to call platform apis in background tasks?
+
+Background tasks running under some consumer or webhook or under any queue can get platform client via method `get_platform_client`. It will return instance of `PlatformClient` as well. 
+
+> Here FdkClient `access_mode` should be **offline**. Cause such client can only access PlatformClient in background task. 
+
+```python
+async def background_handler(request):
+    try:
+        company_id = request.args.get("companyId")
+        client = await fdk_extension_client.get_platform_client(company_id)
+        return response.json({"success": True})
+    except Exception as e:
+        return response.json({"error_message": str(e)}, 500)
+```
+
+
+#### How to register for webhook events?
+
+Webhook events can be helpful to handle tasks when certan events occur on platform. You can subscribe to such events by passing webhook_config in setupFdk function.
+
+```python
+fdk_extension_client = setup_fdk({
+    "api_key": "<API_KEY>",
+    "api_secret": "<API_SECRET>",
+    "base_url": base_url,
+    "scopes": ["company"],
+    "callbacks": extension_handler,
+    "storage": redis_connection,
+    "access_mode": "offline",
+    "cluster": "https://api.fyndx1.de",  # this is optional by default it points to prod.
+    "webhook_config": {
+    "api_path": "/api/v1/webhooks", # required
+    "notification_email": "test@abc.com", # required
+    "subscribe_on_install": False, # optional. Default true
+    "subscribed_saleschannel": "specific", #optional. Default all
+    "event_map": {  # required
+      'brand/create': {
+        "version": '1',
+        "handler": handleBrandCreate
+      },
+      'extension/uninstall': {
```

Comment:
Update example as per this ref: https://github.com/gofynd/fdk-extension-javascript/blob/main/README.md

Reply:
Example Updated